app/components/work_package_relations_tab/index_component.rb

```diff
@@ -11,13 +11,14 @@ class WorkPackageRelationsTab::IndexComponent < ApplicationComponent
 
   attr_reader :work_package, :relations, :children, :directionally_aware_grouped_relations
```

Comment:
Please add the attribute here as well so that you can access it more easily in the template